django_summernote/templates/django_summernote/widget_common.html

```diff
@@ -93,6 +93,11 @@
                     {% if not config.disable_attachment %}
                     onImageUpload: function(files) {
                         // custom attachment data
+                        if ($(".note-editable")[0]){
+                            var s = '<img id="loader" src="https://media3.giphy.com/media/3oEjI6SIIHBdRxXI40/giphy.gif">';
+
+                            $(".note-editable")[0].innerHTML = s + $(".note-editable")[0].innerHTML
+                        }
```

Comment:
@cmaliwal-ergeon what about other cases? i.e. videos or other attachments?

Reply:
@chapayevdauren For videos, we insert links, so basically they are displayed directly from their original links and not uploaded.
And other options are hyperlinks.

Reply:
@chapayevdauren For videos, we insert links, so basically they are displayed directly from their original links and not uploaded.
And other options are hyperlinks.